apps/console/src/features/sms-providers/pages/sms-providers.tsx

```diff
@@ -0,0 +1,712 @@
+                                                                            showSetupGuideButton={ false }
+                                                                            showCardAction={ false }
+                                                                        />
+                                                                    </Grid.Column>
+                                                                )) }
+                                                        </Grid.Row>
+                                                    </Grid>
+                                                </div>
+                                                { smsProviderSettings.selectedProvider && (
+                                                    <div>
+                                                        <Divider hidden />
+                                                    </div>
+                                                ) }
+                                                { smsProviderSettings.selectedProvider && (
+                                                    <div>
+                                                        { smsProviderSettings.selectedProvider === 
+                                                        SMSProviderConstants.CUSTOM_SMS_PROVIDER && (
+                                                            <CustomSMSProvider
+                                                                isReadOnly={ isReadOnly }
+                                                                onSubmit={ handleSubmit }
+                                                            />
+                                                        ) }
+                                                        { smsProviderSettings.selectedProvider === 
+                                                        SMSProviderConstants.TWILIO_SMS_PROVIDER && (
+                                                            <TwilioSMSProvider
+                                                                isReadOnly={ isReadOnly }
+                                                                onSubmit={ handleSubmit }
+                                                            />
+                                                        ) }
+                                                        { smsProviderSettings.selectedProvider === 
+                                                        SMSProviderConstants.VONAGE_SMS_PROVIDER && (
+                                                            <VonageSMSProvider
+                                                                isReadOnly={ isReadOnly }
+                                                                onSubmit={ handleSubmit }
+                                                            />
+                                                        ) }
+                                                    </div>
```

Comment:
Let's use a fragment instead of a `<div>`
```suggestion
                                                    <>
                                                        { smsProviderSettings.selectedProvider === 
                                                        SMSProviderConstants.CUSTOM_SMS_PROVIDER && (
                                                            <CustomSMSProvider
                                                                isReadOnly={ isReadOnly }
                                                                onSubmit={ handleSubmit }
                                                            />
                                                        ) }
                                                        { smsProviderSettings.selectedProvider === 
                                                        SMSProviderConstants.TWILIO_SMS_PROVIDER && (
                                                            <TwilioSMSProvider
                                                                isReadOnly={ isReadOnly }
                                                                onSubmit={ handleSubmit }
                                                            />
                                                        ) }
                                                        { smsProviderSettings.selectedProvider === 
                                                        SMSProviderConstants.VONAGE_SMS_PROVIDER && (
                                                            <VonageSMSProvider
                                                                isReadOnly={ isReadOnly }
                                                                onSubmit={ handleSubmit }
                                                            />
                                                        ) }
                                                    </>
```

Reply:
fixed: 6494aac1141609560b62f9b1294241d37ef00c95